sdks/python/apache_beam/examples/snippets/transforms/aggregation/combineglobally.py

```diff
@@ -0,0 +1,214 @@
+# coding=utf-8
```

Comment:
Consider sharing more code among the other combine PR and GBK PR.

Reply:
I changed the examples for the Combine* transforms and GBK. Hopefully, they make more sense now.